packages/react-server-cli/src/compileClient.js

```diff
@@ -159,9 +160,13 @@ function statsToManifest(stats) {
 	};
 }
 
-function packageCodeForBrowser(entrypoints, outputDir, outputUrl, hot, minify, longTermCaching, stats) {
+function packageCodeForBrowser(entrypoints, outputDir, outputUrl, hot, minify, longTermCaching, stats, configPath) {
 	const NonCachingExtractTextLoader = path.join(__dirname, "./NonCachingExtractTextLoader");
 	const extractTextLoader = require.resolve(NonCachingExtractTextLoader) + "?{remove:true}!css-loader";
+	const babelrcPath = path.join(configPath || '', ".babelrc");
+	const babelConfig = configPath && fs.existsSync(babelrcPath) ?
```

Comment:
Hm, allowing Babel, ESLint, etc config in `package.json` is a really important feature here, and feels like it is getting into deep complexity.

I think for now I'll make a note in the description above and close this pull request, because:
- less configuration is better
- there's not a concrete use case for this right now
- this may create weird edge case bugs